Aptoide/bds-sdk
# [APPC-1634] Pay as a guest - Request Wallet/Entry Screen

**What does this PR do?**

Implementation of the wallet generation for pay as a guest
Implementation of the entry screen for pay as a guest

**Database changed?**

No

**Where should the reviewer start?**

PaymentMethodsFragment.java

**How should this be manually tested?**

Check if the wallet is created and check if when you request with same id the wallet is the same
Enter the screen and test the flow

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1634

**Code Review Checklist**
